tests/javascript/index.php

```diff
@@ -5355,6 +5361,40 @@ function testWithWrappedConsole(callback) {
         }, 1500);
     });
 
+    test("Test API - enable/disable CampaignParameters", function() {
+        expect(5);
```

Comment:
It's beyond me why there's 5 assertions in this test when we explicitly call just 2 but there might be some further ones called by some of the methods somehow? I think we can leave it as is unless someone wants to really dig deep into it.

Reply:
I expect those three extra expectations to come from the `module("request")` setup/teardown expectations. Not checked all tests, but most seem to have 3 more assertions exected than used itself, matching the module code.